### DISCUSSION TITLE
please help :D

### DISCUSSION THREAD

**Discussion by qdsgokjerk** [go-vgo/robotgo#580] [Q&A] [answered]
Hello, I am very weak in programming and I do not understand how to fix these errors. Can someone help or has anyone encountered the same problem?
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:51:10: undefined: addEvent
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:62:7: undefined: Start
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:75:18: undefined: KeyHold
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:83:18: undefined: KeyUp
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:95:22: undefined: KeyUp
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:96:4: undefined: End
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:113:7: undefined: Start
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:121:17: undefined: MouseMove
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:128:22: undefined: MouseDown
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:129:4: undefined: End
C:\Users\Usser\go\pkg\mod\github.com\robotn\gohook@v0.40.0\event.go:129:4: too many errors

**Comment by hektorinho** [comment, marked as answer]
I got the same error when importing the package...

Is there a prior version that works?

Thanks,
--Hektor

**Reply by qdsgokjerk** [reply]
Upd:installation and additions to system variables seem to have been fixed
https://www.youtube.com/watch?v=OwQobefF-iE&t=252s
